discovery-provider/src/api/v1/helpers.py

```diff
@@ -584,13 +589,42 @@ def add_auth_headers_to_parser(parser):
 user_tracks_library_parser.add_argument(
     "type",
     required=False,
-    description="The type of tracks to return: favorited, reposted, purchased, or all. Defaults to favorite",
+    description="The type of entity to return: favorited, reposted, purchased, or all. Defaults to favorite",
     type=str,
     choices=LibraryFilterType._member_names_,
     default=LibraryFilterType.favorite,
 )
 add_auth_headers_to_parser(user_tracks_library_parser)
 
+user_collections_library_parser = pagination_with_current_user_parser.copy()
+user_collections_library_parser.remove_argument("current_user")
+user_collections_library_parser.add_argument(
+    "query", required=False, description="The filter query"
+)
+user_collections_library_parser.add_argument(
+    "sort_method",
+    required=False,
+    description="The sort method",
+    type=str,
+    choices=LimitedSortMethod._member_names_,
+)
+user_collections_library_parser.add_argument(
```

Comment:
Feels like we should have a helper method for adding the sort args at this point, like @nicoback2  did for Auth

Reply:
Good idea, will add